src/encoded/audit/experiment.py

```diff
@@ -1864,29 +1858,40 @@ def audit_experiment_consistent_sequencing_runs(value, system, files_structure):
                     yield AuditFailure('mixed read lengths',
                                        detail, level='WARNING')
 
-    keys = list(replicate_pairing_statuses.keys())
-    if len(keys) > 1:
-        for index_i in range(len(keys)):
-            for index_j in range(index_i + 1, len(keys)):
-                i_pairs = replicate_pairing_statuses[keys[index_i]]
-                j_pairs = replicate_pairing_statuses[keys[index_j]]
-                diff_flag = False
-                for entry in i_pairs:
-                    if entry not in j_pairs:
-                        diff_flag = True
-                for entry in j_pairs:
-                    if entry not in i_pairs:
-                        diff_flag = True
-                if diff_flag is True:
-                    detail = 'Biological replicate {} '.format(keys[index_i]) + \
-                             'in experiment {} '.format(value['@id']) + \
-                             'has endedness {} '.format(i_pairs) + \
-                             ' that differ from replicate {},'.format(keys[index_j]) + \
-                             ' which has {}.'.format(j_pairs)
-                    yield AuditFailure('mixed run types',
-                                       detail, level='WARNING')
-
-    return
+    # run type consistency is relevant only for ChIP-seq
+    if assay_term_name == 'ChIP-seq':  
```

Comment:
We shouldn't need to check this.  replicate_pairing_statuses will be empty if assay_term_name is not 'ChIP-seq'.  If we don't need this check then I believe we don't need to rearrange the code blocks.  

I suggest putting the code back to it's original state with the three line changes at the top of the file. 

Rearranging the code blocks adds a bunch of work to what could be a three line change.  If we keep the rearrangement then we'll need to fix the pep8 errors, string formatting, and there is probably a better way to write those nested for loops.  When working in preexisting files make as few changes as possible because it is easier and safer.  Lines of code that are changed need to be reviewed and fixed if needed.